ServerCore/Pages/Teams/Hints.cshtml.cs

```diff
@@ -15,6 +15,8 @@ namespace ServerCore.Pages.Teams
     [Authorize(Policy = "PlayerIsOnTeam")]
     public class HintsModel : EventSpecificPageModel
     {
+        private static bool IsBeta = true;  // TODO: I don't know how to tell if this is a beta; this code seems not to have the event object available
```

Comment:
EventSpecificPageModel has an Event property on it. I think you can use Event.Name.Contains("Beta") like you wanted.